### PR TITLE
WF - added the wildfire-get-url-webartifacts cmd and report more outputs

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] In Hold - WF open issues(empty folders)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34148

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] add `wildfire-get-url-webartifacts`
- [x] add detection outptus to the `wildfire-report` cmd
- [x] Add it to the `WF URL2.1 detonate URL PB` -> added the outputs, webartifacts tar parsing was moved to be discussed in a separate issue.
- [x] Tests
- [x] Documentation - after @amshamah419 review of the outputs desc

### CODE REVIEW

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 1]
```diff
@@ -296,6 +295,27 @@ script:
...
+      description: Name of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.type
+      description: Type of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.verdict
+      description: Verdict of the detection.
+      type: string
+    - contextPath: WildFire.Report.detection_reasons.artifacts
+      description: Artifacts of the detection reasons.
```
Comment (amshamah419) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:311]:
```suggestion
      description: Artifacts for the detection reasons.
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 2]
```diff
@@ -415,7 +435,28 @@ script:
...
+  - arguments:
+    - default: false
+      description: URL of the webpage.
+      isArray: true
+      name: url
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether to download screenshots or as downloaded files.
```
Comment (amshamah419) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:447]:
```suggestion
      description: Whether to download as screenshots or as downloadable files.
```

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_3_0.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### Palo Alto Networks WildFire v2
+- Added the ***wildfire-get-url-webartifacts*** command.
+- Added the *detection_reasons* to the outputs of the ***wildfire-report*** command.
```
Comment (amshamah419) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_3_0.md:5]:
```suggestion
- Added *detection_reasons* to the outputs of the ***wildfire-report*** command.
```

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_3_0.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+- Upgraded the Docker image to: *demisto/python3:3.9.2.17246*.
+
+#### Playbooks
+##### Detonate URL - WildFire v2.1
+- Added the *detection_reasons* to the playbook outputs.
```
Comment (amshamah419) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_3_0.md:10]:
```suggestion
- Added *detection_reasons* to the playbook outputs.
```

**Comment by bakatzir** [comment]
@yuvalbenshalom Added a README in the new format.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**